packages/i18n-calypso/package.json

```diff
@@ -40,7 +40,7 @@
 	},
 	"scripts": {
 		"clean": "rm -rf dist",
-		"build": "transpile",
+		"build": "IMPORT_SOURCE=react run -T transpile",
```

Comment:
A bummer we didn't have it the other way around in the beginning - set to `@emotion/react` where needed. `@babel/preset-react` will use `react` as an `importSource` by default, right?

I wonder where we really need to use `@emotion/react` and if we can specifically enable it there only while relying on the default `react` one everywhere else.

Reply:
> I wonder where we really need to use `@emotion/react` and if we can specifically enable it there only while relying on the default `react` one everywhere else.

I think we can check which packages import from `@emotion` and assume that these packages should use `@emotion/react` as an `importSource`.

I'll look more into detail and see if we can safely change that. However, it will likely be in a separate PR.

Reply:
Sounds good, and a separate PR is fine 👍